src/ui.js

```diff
@@ -502,6 +502,18 @@ UIMode.Command = class extends UIMode {
     }
 };
 
+// We are viewing a diagram embedded in another page.
+UIMode.Embed = class extends UIMode {
+    constructor(ui) {
+        super();
+
+        this.name = "embed";
+
+        ui.grid.class_list.toggle("hidden");
+        ui.pan_view(Offset.zero(), -1.0);
```

Comment:
I think it'd be preferable to add a query parameter `scale` that sets the initial zoom level (and works in normal or embedded mode), which gives a little more flexibility.